packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -260,7 +259,6 @@ const ComboBox = React.forwardRef((props: ComboBoxProps, ref) => {
     ['aria-label']: ariaLabel,
     ariaLabel: deprecatedAriaLabel,
     className: containerClassName,
-    direction,
```

Comment:
Comment from author: I removed this line to satisfy linter (unused variable), but if might be a bigger problem - the entire `direction` prop unused / not working in v11. Please review this one with special attention.

Reply:
Interesting, it seems to be working [here](https://react.carbondesignsystem.com/?path=/story/components-combobox--playground&args=direction:top).

I think the issue is that just the `[enabled ? null : containerClassName],` line needs to be removed, not the entire `className` block.

Reply:
Good finding! I pushed a commit with partial revert.